src/components/LoansListItem.tsx

```diff
@@ -32,48 +45,20 @@ const LoanName = styled.h3`
   line-height: 1.2;
   margin-bottom: 10px;
 `;
-
-const Button = styled.button`
-  width: 150px;
-  height: 50px;
-  border-radius: 3px;
-  border: none;
-  cursor: pointer;
-  font-weight: 600;
-  font-size: 24px;
-
-  color: gray;
-  background: #ffff99;
-  text-transform: uppercase;
-  transition: all 150ms linear;
-  &:hover,
-  &:active {
-    background: #ffff00;
-    color: #0099cc;
-  }
-`;
-
-// "title": "Voluptate et sed tempora qui quisquam.",
-//       "tranche": "A",
-//       "available": "11,959",
-//       "annualised_return": "8.60",
-//       "term_remaining": "864000",
-//       "ltv": "48.80",
-//       "amount": "85,754"
-
-interface Loan {
+export interface Loan {
   id: String;
   title: String;
   available: String;
   annualised_return: String;
   term_remaining: String;
   ltv: String;
   amount: String;
+  invested: Boolean;
```

Comment:
you add a new field just because you need to show it on UI. It's better to store this field separately